tests/tests.js

```diff
@@ -589,10 +589,8 @@ exports.defineAutoTests = function() {
                 removeContactsByFields(["note"], "DeleteMe", function() {
                     contact.save(function(c_obj) {
                         var findWin = function(cs) {
-                            // update to have proper saved id
-                            contact = cs[0];
                             expect(cs.length).toBe(1);
-                            done();
+                            done(cs[0]);
```

Comment:
never mind. I realized ```done``` here is bound to a different function. can we rename this function to ```callback``` or something else ? It's confusing.